### PR TITLE
Port over Scholar nav bar

### LINKED ISSUE #273: Replicate Scholar's navbar header
### Descriptive summary

Change the navbar header in the upper-left corner of the screen to match Scholar.  See the areas highlighted in red below.

You will need to override Hyrax's app/views/_masthead.html.erb and app/views/_logo.html.erb partials.  Find them in Scholar's repo to see how we changed them.

You will also need to make the entire top bar have a black background and a height that matches Scholar's

On the home page:
home

On all pages except the home page:
browse

### PR DESCRIPTION
**Pull request by crowesn**
Fixes #273 

Port over navbar style from scholar@uc.

Changes proposed in this pull request:
* Add logo assets
* Add styles

### CODE REVIEW

**Comment by hortongn** [comment]
@crowesn The feature spec errors you are seeing are legitimate and specific to your branch.  I can replicate the errors consistently just on your branch.

One of your failing specs is `spec/features/create_image_spec.rb:36`

Here is what capybara sees on the `develop` branch for that (passing) test:

screenshot copy

and here is what capybara sees on your branch for the same (failing) test:

screenshot

Things are a little off with the layout and it's somehow throwing off capybara from seeing what it expects to see.  That's why all your failures tend to be on create work or edit work pages.  You'll see the same glitch in your local environment if you start the app and navigate to a new work form.

**Comment by crowesn** [comment]
@hortongn This should be good to go.

File: spec/features/collection_type_spec.rb [hunk 1]
```diff
@@ -383,7 +383,7 @@
 
           sleep 5
 
-          find(:xpath, "/html/body/div[2]/div[2]/div[2]/div[3]/div/div/div[2]/a", text: 'View collections of this type').click
+          find(:xpath, "/html/body/div[3]/div[2]/div[2]/div[3]/div/div/div[2]/a", text: 'View collections of this type').click
```
Comment (hortongn) [inline spec/features/collection_type_spec.rb]: This spec has been fixed in #326, but it hasn't been merged yet.  Just take it out of your PR and I'll merge even if this spec fails.

**Comment by crowesn** [comment]
@hortongn change requests fulfilled. Thanks!

**Pull request merged**